content/news/2023-06-24-elixir-annual-report-2022/index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: 'ELIXIR releases 2022 Annual Report - the Galaxy perspective'
+date: "2023-11-17"
```

Comment:
Shouldn't this be also June or something?

Reply:
This was exactly my question (be strict or enhance visibility)

Reply:
Visibility is gone, this is old news. We have been to slow. So this is about linking in reports and getting a (hopefully comprehensive) overview about the year imho.

Reply:
Ok - has been changed. I hope for a sufficient degree of comprehensiveness, too.